nautobot/extras/api/serializers.py

```diff
@@ -560,6 +560,8 @@ def validate(self, data):
 
 
 class JobQueueSerializer(NautobotModelSerializer, TaggedModelSerializerMixin):
+    display_with_worker_count = serializers.CharField(read_only=True)
```

Comment:
Seems like a weird property to render as a string in the REST API - what's the intent here?

Reply:
Will just replace it with `display`